FLEDGE.md

```diff
@@ -803,3 +811,184 @@ These signals were requested in [issue 435](https://github.com/WICG/turtledove/i
+
+If an additional bid only needs to specify a single negative interest group, it can do so as follows:
+
+```
+const additionalBid = {
+  // ...
+  "negativeInterestGroup": "example_advertiser_negative_interest_group"
+  // ...
+}
+```
+
+If an additional bid needs to specify two or more negative interest groups, all of those negative interest groups must be joined from the same site, and that site must be identified ahead of time in the additional bid using the joiningOrigin field:
+
+```
+const additionalBid = {
+  // ...
+  "negativeInterestGroups": {
+    joiningOrigin: "https://example-advertiser.com",
+    interestGroupNames: [
+      "example_advertiser_negative_interest_group_a",
+      "example_advertiser_negative_interest_group_b",
+    ]
+  },
+  // ...
+}
+```
+
+Any negative interest group that wasn't joined from that identified site won't be considered for negative targeting. This restriction is enforced so that negative targeting can only use targeting data from a single site.  An additional bid that only specifies one negative interest group is not subject to the same restriction on joining origin.
+
+##### 6.2.3 Additional Bid Keys
+
+We use a cryptographic signature mechanism to ensure that only the owner of a negative interest group can use it with additional bids. Each buyer will need to create a Ed25519 public/secret key pair to sign their additional bids to prove their authenticity, and to regularly rotate their key pairs.
+
+When a buyer joins a user into a negative interest group, they must provide their 32-byte Ed25519 public key, expressed as a base64-encoded string, via the negative interest group's additionalBidKey field. This can be seen in the example above in section 6.2.1 Negative Interest Groups. The additionalBidKey can be then updated via the negative interest group's updateURL, for example, to enable a buyer to rotate their Ed25519 key pair faster than they could with the expiration of their negative interest groups alone.
+
+When the buyer issues an additional bid, that bid needs to be signed using their Ed25519 secret key. During a key rotation, the buyer may need to provide a signature of the additional bid with both the old and the new additionalBidKeys while negative interest groups stored on users' devices are updated to the new key. It's for this reason that additional bids may have more than one signature provided alongside the bid.
+
+If the signature doesn't verify successfully, the additional bid proceeds as if the negative interest group is not present.  This "failing open" ensures that only the owner of the negative interest group, who created the additonalBidKey, is allowed to negatively target the interest group, and that nobody else can learn whether the interest group is present on the device.  Because the signature check "fails open", buyers should make sure they're using the right keys; for example it might be pertinent to verify a bid signature before submitting the additional bid.
```

Comment:
>signature doesn't verify successfully, the additional bid proceeds as if the negative interest group is not present

I suppose this implicitly answers the earlier quesiton about whether an additional bid may not specify any negative IG info.